src/Compilers/CSharp/Test/Emit/PDB/PDBUsingTests.cs

```diff
@@ -525,10 +525,10 @@ class A { void M() {  } }
                 options: TestOptions.DebugDll,
                 references: new[]
                 {
-                    // first aliased reference
-                    compilation1.ToMetadataReference(ImmutableArray.Create("X")),
-                    // second unaliased reference 
-                    compilation1.ToMetadataReference(),
+                // first aliased reference
```

Comment:
This change is a result of the `Formatter.Format` call which is used without any special options.  I'm not an expert in this area but it does seem like a bug to me.  I would expect it to indent one level here.